keras/utils/np_utils.py

```diff
@@ -48,3 +49,62 @@ def normalize(x, axis=-1, order=2):
     l2 = np.atleast_1d(np.linalg.norm(x, order, axis))
     l2[l2 == 0] = 1
     return x / np.expand_dims(l2, axis)
+
+def mxnet_image_channels_conversion(np_data):
```

Comment:
* Why should we call it mxnet_image_channels_conversion?
API should be independent of a framework, it is just transposing an image. Please rename and make it generic.

* As we discussed earlier, it should take both single np_array and list of np_array. Think like a user, he loads a batch of data, and he will call this API to transpose.

May be below API?
```
def to_channels_first(data):
```

Reply:
- Initially, I thought it's just for MXNet backend and I have coded that way. But your point makes more sense to have it generic and not bind to anyone backends. I have modified the code and now it's more generic across backends.
- Absolutely. Handled the case where a user can now provide either a Numpy tensor or an array of Numpy tensor.
- Thank-You.